### PR TITLE
Added Unique Signifier Model and Voice

### PR DESCRIPTION
**Pull request by Subsourian**
Adds a unique model for the Signifier, commissioned for Archipelago and created by Enoki, so they will no longer be the generic High Templar model. Adds voicework, models, animations, buttons, and effects for the Signifier. Adds an MIT license for use of the model from the owner, and updates the readme to reflect the custom asset and give credit. All changes are visual, the units stats remain the same as they were.

image

### CODE REVIEW

**Comment by MadiMadsen** [comment]
OHHHHH I LOVE THIS 10/10 work

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/BehaviorData.xml [hunk 1]
```diff
@@ -36,6 +36,16 @@
             <ModifyFlags index="SuppressMoving" value="1"/>
             <ModifyFlags index="SuppressTurning" value="1"/>
         </Modification>
+    </CBehaviorBuff>
+        <CBehaviorBuff id="PsiStormShakuras">
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/BehaviorData.xml:40]: indent, id should start with `AP_` prefix

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml [hunk 2]
```diff
@@ -13242,10 +13262,11 @@
     <CActorSound id="AP_HighTemplar_Death" parent="SoundOneShot">
         <On Terms="UnitDeathCustomize.AP_HighTemplar; ValidateUnit IsNotHallucination; PassChance 25.000000" Send="Create"/>
         <On Terms="UnitDeathCustomize.AP_HighTemplarAir; ValidateUnit IsNotHallucination; PassChance 25.000000" Send="Create"/>
-        <On Terms="UnitDeathCustomize.AP_HighTemplarShakuras; ValidateUnit IsNotHallucination; PassChance 25.000000" Send="Create"/>
-        <On Terms="UnitDeathCustomize.AP_HighTemplarShakuras; ValidateUnit IsNotHallucination; PassChance 25.000000" Send="Create"/>
-        <On Terms="UnitDeathCustomize.AP_HighTemplarShakuras; ValidateUnit IsNotHallucination; PassChance 25.000000" Send="Create"/>
         <Sound value="HighTemplar_Death"/>
+    </CActorSound>
+        <CActorSound id="HighTemplarShakuras_Death" parent="SoundOneShot">
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml:13267]: Indent

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/SoundData.xml [hunk 3]
```diff
@@ -9030,6 +9030,75 @@
...
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes00.wav"/>
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes01.wav"/>
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes02.wav"/>
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes03.wav"/>
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes04.wav"/>
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes05.wav"/>
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes06.wav"/>
+        <AssetArray File="AP\LocalizedData\Sounds\protossvo\campaignunits\HighTemplarNerazim\Signifier_Yes07.wav"/>
+        <Volume value="2.000000,2.000000"/>
+    </CSound>
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/SoundData.xml:9101]: `AP_` prefix

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml [hunk 4]
```diff
@@ -12831,6 +12832,22 @@
         <AttachProps Id="Ref_Origin" SquibType="Shield" RadiusShield="2.750000">
             <Keys index="Shield" value="1"/>
         </AttachProps>
+    </CModel>
+        <CModel id="HighTemplarShakurasDeathRagdoll" parent="HighTemplarDeathRagdollBase">
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml:12836]: indent

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml [hunk 5]
```diff
@@ -12831,6 +12832,22 @@
...
+        <EditorCategories value="Race:Protoss"/>
+        <Radius value="1.000000"/>
+    </CModel>
+    <CModel id="HighTemplarShakurasPsiStormModel" parent="PersistentSpellFX">
+        <Model value="AP\Assets\Effects\Protoss\SignifierCripple\PsiStorm_Cripple.m3"/>
+        <EditorCategories value="Race:Protoss"/>
+        <RadiusLoose value="1.000000"/>
+    </CModel>
+    <CModel id="HighTemplarShakurasShadow" parent="HighTemplarShadowBase">
+        <Model value="AP\Assets\Units\Protoss\HighTemplarNerazim\HighTemplar_Nerazim_ShadowTrail.m3"/>
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ModelData.xml:12850]: `AP_` Prefix

File: Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml [hunk 6]
```diff
@@ -1015,6 +1015,22 @@
...
+        <HostSiteOps Ops="SOpAttachCenter"/>
+        <Model value="HighTemplarPsiStormImpact"/>
+        <AutoScaleFactor value="0.750000"/>
+    </CActorModel>
+    <CActorModel id="HighTemplarShakurasPsiStormModel" parent="ModelAnimationStyleContinuous">
+        <On Terms="Effect.AP_PsiStormShakurasPersistent.Start" Send="Create"/>
+        <On Terms="Effect.AP_PsiStormShakurasPersistent.Stop" Send="AnimBracketStop BSD"/>
+        <HostSiteOps Ops="SOpShadow"/>
+        <Scale value="1.300000"/>
+    </CActorModel>
```
Comment (Ziktofel) [inline Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml:1033]: `AP_` Prefix

**Review by Ziktofel — CHANGES_REQUESTED** [review]
Add the prefix and fix the intentation

**Pull request merged**